source/mainnet/net/voting/voting.rst

```diff
@@ -118,6 +120,20 @@ These older wallets do not have the capability to connect to dApps. But you do h
 
    #. It is possible to check from the Voting DApp that the delegation worked. The window for delegating votes is the same as voting with the DApp.
 
+.. dropdown:: Concordium Client
+
+    Users need to create a new account in |bw| or |cryptox| and delegate their vote from their Concordium Client wallet account(s) to the new account, then vote from the new account. You can create the new account in |bw| or |cryptox| at any time before the election.
+
+    **To delegate your vote:**
+
+    .. code-block:: console
+
+        $concordium-client transaction send --amount AMOUNT --receiver A --sender B --memo delegatevote2024
```

Comment:
I have checked the command and indeed I receive tx with the correct memo. However, notice that this command works only if user has a local node running.

For better usability, we would add --grpc-ip grpc.mainnet.concordium.software so that they can just use grpc node. Also, on that node we have TLS so --secure flag needs to be used. So:
`concordium-client --secure --grpc-ip grpc.mainnet.concordium.software transaction send --amount AMOUNT --receiver A --sender B --memo delegatevote2024`